ci-operator/step-registry/addon/install/addon-install-commands.sh

```diff
@@ -8,13 +8,13 @@ set -o verbose
 CLUSTER_NAME=$(cat "${SHARED_DIR}/cluster-name")
 OCM_TOKEN=$(cat /var/run/secrets/ci.openshift.io/cluster-profile/ocm-token)
 BREW_TOKEN=$(cat /var/run/secrets/ci.openshift.io/cluster-profile/brew-token)
-RUN_COMMAND="poetry run python app/cli.py addon --cluster ${CLUSTER_NAME} --token ${OCM_TOKEN} --api-host ${API_HOST} --timeout ${TIMEOUT} "
+RUN_COMMAND="poetry run python app/cli.py addons --cluster ${CLUSTER_NAME} --token ${OCM_TOKEN} --api-host ${API_HOST} --timeout ${TIMEOUT} "
```

Comment:
`--timeout ${TIMEOUT}` is unnecessary since if addon doesn't indicate it's OWN timeout, [TIMEOUT_30MIN](https://github.com/RedHatQE/ocp-addons-operators-cli/pull/115/files#diff-a37a03277bb3016cc33210e3f7e32179812e8d7d65040a95e0bf7ec8924dfdfbR43) will be taken